marklogic-data-hub-central/ui/src/components/query-export/query-export-modal/query-export-modal.tsx

```diff
@@ -120,6 +127,15 @@ const QueryExportModal = (props) => {
             </div>
             }
         </Form>
+
+        {props.hasStructured && <div>
+            <Collapse bordered={false} className={styles.collapseCustomPanel} style={{ border: '0px', color: 'blue' }}>
+                <Panel id='export-panel' header="Show Preview" key="1" style={{ border: '0px', color: 'blue' }}>
+                    <Table data-testid='export-preview-table'className={styles.exportTable} dataSource={props.tableData} columns={props.tableColumns} pagination={false} size='small' scroll={{ x: 500 }} bordered />
+                </Panel>
+            </Collapse>
+        </div>}
+
     </Modal>
 
     return (
```

Comment:
no reason to create the exportConfirmation variable. just return the JSX.

Reply:
Changed. Thanks.